package.json

```diff
@@ -63,7 +64,7 @@
     "storybook-formik": "^2.4.2",
     "styled-components": "^5.3.5",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.2",
+    "typescript": "^5.0.4",
```

Comment:
Not a PR question: is there any reason to setup typescript as a prod dependency? There're recommendations to install it as a dev dependency https://www.typescriptlang.org/download
<img width="543" alt="image" src="https://github.com/fs/react-ts-next-base/assets/46020998/ef312111-1b4d-4381-8776-8d9823edcdc2">

Reply:
I agree, no reason. Fixes